codesydney/codesydney
# #22 Home Page Member Profile Widget

## Rationale
Display Code.Sydney members at the homepage, as an overview of members' profiles. 

## Advice for Reviewers & Testing Notes
- **Open one terminal**
- ```cd client```
- ``` npm install``` to make sure node-sass, sass-loader, axios, font awesome, and json-server work properly.
- ``` npm run serve```

- **Open another terminal**
- ```cd client```
- run ```json-server --watch membersDb.json``` to watch local json file for axios
- Go to your browser and navigate to ```localhost:8080```

- Note: Save members' data in membersDb.json temporarily. Need run ```json-server --watch membersDb.json``` to fetch data. The baseURL for axios will be replaced when the backend is ready.

## Screenshots:

- Add a screenshot or two of your changes, where applicable.

![image](https://user-images.githubusercontent.com/53432888/88051549-2687eb80-cb9c-11ea-9345-f7208254a180.png)
![image](https://user-images.githubusercontent.com/53432888/88025851-47d6e080-cb78-11ea-9aa5-0bb68109e621.png)
![image](https://user-images.githubusercontent.com/53432888/88025863-4f968500-cb78-11ea-922b-397d146d7d41.png)


## Task Name

- #22 Homepage-member profile widget


## Linting Checklist
